copyright

```diff
@@ -2285,7 +2276,7 @@ Copyright: Daeridanii (https://github.com/Daeridanii1)
 License: CC0
```

Comment:
Does "CCO" require the same clarification as "public-domain"?

Reply:
Not according to the specification; `public-domain` is a special entry. As far as I know, CC0 is just a really permissive license, not a special thing.


> The License short name public-domain does not refer to a set of license terms. There are some works which are not subject to copyright in any jurisdiction and therefore no license is required for any purpose covered by copyright law. This short name is an explicit declaration that the associated files are “in the public domain”.

Reply:
I do think we are allowed to omit the CC0 license and replace it with public-domain, though, that seems to be a CC0 feature. Not that I see a reason to do that.